setup.cfg

```diff
@@ -68,3 +68,6 @@ ignore_missing_imports = True
 
 [mypy-setuptools]
 ignore_missing_imports = True
+
+[mypy-xml.*]
```

Comment:
I tried running `pytest --mypy` without this and I didn't see any errors. Is this needed?

Reply:
I was getting this error w/o it, so it was added:

```bash
➜  colcon-sanitizer-reports git:(master) ✗ mypy .
colcon_sanitizer_reports/xml_output_generator.py:17: error: No library stub file for standard library module 'xml.dom.minidom'
colcon_sanitizer_reports/xml_output_generator.py:17: note: (Stub files are from https://github.com/python/typeshed)
colcon_sanitizer_reports/xml_output_generator.py:17: error: No library stub file for standard library module 'xml.dom'
```